code-challenges.js

```diff
@@ -21,48 +21,111 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
-
+//Function Signature
+//input: number
+//output: string indicating greater, less than, or equal to
+//
+//input: 42
+//output: 42 is below boiling point
+//
+// input: 350
+// output: 350 is above boiling point
+//
+// input: 212
+// output: 212 is at boiling point
+// Process: Create a function called boilingTemp that takes in a number called temp.
+
+const boilingTemp = (temp) => {
+  // If temp is greater than 212, return "temp is over the boiling point.
+  if (temp >212) {
+    return `${temp} is above boiling point`
+  }
+//If temp is less than 212, return "temp is below boiling point.
+  else if (temp<212){
+    return `${temp} is below boiling point`
+  }
+  //If temp is 212, return "temp is at boiling point".
+  else if (temp===212){
+    return `${temp} is at boiling point`
+  }
+  //Any other input, return error.
+  else {
+    return "error"
+  }
+}
 const temp1 = 42
+console.log(boilingTemp(42))
 // Expected output: "42 is below boiling point"
+//Actual output as expected.
 
 const temp2 = 350
+console.log(boilingTemp(350))
 // Expected output: "350 is above boiling point"
+//Actual output as expected.
 
 const temp3 = 212
+console.log(boilingTemp(212))
 // Expected output: "212 is at boiling point"
+//Actual output as expected.
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: Create one array named padres1984WorldSeriesRuns and one array called padres1998WorldSeriesRuns with the number of runs from each of those world series.
+//Create a variable called comboRuns that combines the two World Series Runs arrays using concat.
+// Show the length of the array using .length
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+const comboRuns = padres1984WorldSeriesRuns.concat (padres1998WorldSeriesRuns)
+console.log (comboRuns.length)
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+// Pseudo code: Create an array from the letters of the string by using split. Then reverse the order of the characters using reverse. Then put them back into a string again.
 
 const currentCohort = "Hotel 2022"
 // Expected output: "2202 letoH"
+// Pseudo code: Input a string
+// Output: a string in reverse
+// "2202 letoH"
+// I need to separate each character and put it into an array and then reverse the array and then recombine the letters into a new string.
+
+const chars = currentCohort.split("");
+console.log(chars[6]);
+// expected output: "l"
+
+const currentCohortCopy = currentCohort.split();
+console.log(currentCohortCopy);
+// expected output: Array ["h o t e l"]
+
+
```

Comment:
looks like this was submitted without completing answer the question. Your pseudocode had a great plan. When you began to code, you created an array but console.log a value from the array instead of the array. Can you explain the process you were taking on lines 96 - 101?